types/openfin/_v2/api/external-window/external-window.d.ts

```diff
@@ -61,7 +60,7 @@ export declare class ExternalWindow extends EmitterBase<ExternalWindowEvents> {
     /**
      * Gives focus to the external window.
      * @return {Promise.<void>}
-     * @emits ExternalWindow#focused
+     * @fires ExternalWindow#focused
```

Comment:
another weird change. I've never heard of either of these tags.

Reply:
Yeah I ended up reverting this back to `master`.